<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATMTransferMoneyForm</class>
 <widget class="QWidget" name="ATMTransferMoneyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_transfer_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transferir</string>
   </property>
  </widget>
  <widget class="Line" name="line_a">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_accept_a">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_return">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>360</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_accept_b">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Verificar</string>
   </property>
  </widget>
  <widget class="Line" name="line_b">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>90</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_a">
    <item>
     <widget class="QLabel" name="label_card_id">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el número de cuenta :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_card_id"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>261</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_b">
    <item>
     <widget class="QLabel" name="label_card_money">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese monto : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_card_money"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>110</y>
     <width>363</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_c">
    <item>
     <widget class="QLabel" name="label_card_info">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>verifique la información de la cuenta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="table_view_card_info"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>reiniciar</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_transfer_money</zorder>
  <zorder>line_a</zorder>
  <zorder>button_accept_a</zorder>
  <zorder>button_return</zorder>
  <zorder>button_accept_b</zorder>
  <zorder>line_b</zorder>
  <zorder>button_reset</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
